cmd/lit-af/shell.go

```diff
@@ -108,6 +108,16 @@ func (lc *litAfClient) Shellparse(cmdslice []string) error {
 		return parseErr(err, "push")
 	}
 
+	if cmd == "add" {
```

Comment:
wonder if `add` / `clear` is not too generic, and should be `addhtlc` and `clearhtlc` in stead.